### PR TITLE
Add back all old emby tests

### PR DESCRIPTION
**Pull request by Bond-009**
I commented out all failing tests for now.
Most of these `Fact`s should be moved to xunit `Theory`s, but that was out of the scope of this PR imo.

### CODE REVIEW

**Comment by EraYaN** [comment]
CI does not seem to run or find the tests though. Not sure why not, does not publish any results either, while that used to work.

File: tests/Jellyfin.Naming.Tests/TV/EpisodePathParserTest.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public void ParseEpisodesCorrectly(string path, string name, int season, int epi
             Assert.Equal(episode, res.EpisodeNumber);
 
             // testing other paths delimeter
-            var res2 = p.Parse(path.Replace('/', '\\'), false);
+            var res2 = p.Parse(path.Replace('/', '/'), false);
```
Comment (Narfinger) [inline tests/Jellyfin.Naming.Tests/TV/EpisodePathParserTest.cs]: I am not sure why this was changed.
The idea behind the original replacement was to include windows and unix directory delimiter in the test cases without having to write both. To me, both regexp seem to have to match the path.
Similar replacement was also done below.

Comment (Bond-009) [reply tests/Jellyfin.Naming.Tests/TV/EpisodePathParserTest.cs]: I did a find+replace for backslashes as they fail when tested on linux.

**Comment by EraYaN** [comment]
Seems to work now

File: Emby.Naming/Common/NamingOptions.cs [hunk 2]
```diff
@@ -176,7 +177,7 @@ public NamingOptions()
 
             CleanDateTimes = new[]
             {
-                @"(.+[^ _\,\.\(\)\[\]\-])[ _\.\(\)\[\]\-]+(19[0-9][0-9]|20[0-1][0-9])([ _\,\.\(\)\[\]\-][^0-9]|$)"
+                @"(.+[^_\,\.\(\)\[\]\-])[_\.\(\)\[\]\-](19[0-9][0-9]|20[0-1][0-9])([ _\,\.\(\)\[\]\-][^0-9]|).*(19[0-9][0-9]|20[0-1][0-9])*"
```
Comment (JustAMan) [inline Emby.Naming/Common/NamingOptions.cs]: why? this ain't any test! stop sneaking unrelated stuff please :D

Comment (dkanada) [reply Emby.Naming/Common/NamingOptions.cs]: I think it was required to make some of the tests pass, which is why it was changed.

Comment (Bond-009) [reply Emby.Naming/Common/NamingOptions.cs]: Dkanada is correct, it’s needed for some tests. Got this from the last Emby.Naming release

Comment (JustAMan) [reply Emby.Naming/Common/NamingOptions.cs]: Which tests are failing?

Comment (Bond-009) [reply Emby.Naming/Common/NamingOptions.cs]:
```cs
        [Fact]
        public void TestMultiEdition3()
        {
            var files = new[]
            {
                @"/movies/The Phantom of the Opera (1925)/The Phantom of the Opera (1925) - 1925 version.mkv",
                @"/movies/The Phantom of the Opera (1925)/The Phantom of the Opera (1925) - 1929 version.mkv"
            };

            var resolver = GetResolver();

            var result = resolver.Resolve(files.Select(i => new FileSystemMetadata
            {
                IsDirectory = false,
                FullName = i

            }).ToList()).ToList();

            Assert.Single(result);
            Assert.Single(result[0].AlternateVersions);
        }
```
This one

Comment (JustAMan) [reply Emby.Naming/Common/NamingOptions.cs]: Care to explain the changes to the regex then? I can see where their results differ, but why did you e.g. removed whitespace from first group?

Comment (Bond-009) [reply Emby.Naming/Common/NamingOptions.cs]: Like I said, I just copied it from the latest Emby.Naming release
